packages/core-data/src/queried-data/reducer.js

```diff
@@ -138,6 +140,37 @@ const queries = flowRight( [
 	);
 } );
 
+/**
+ * Reducer tracking queries state.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Next state.
+ */
+const queries = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case 'RECEIVE_ITEMS':
+			return receiveQueries( state, action );
+		case 'REMOVE_ITEMS':
+			const newState = { ...state };
+			const removedItems = action.itemIds.reduce( ( result, itemId ) => {
+				result[ itemId ] = true;
+				return result;
+			}, {} );
+			forEach( newState, ( queryItems, key ) => {
+				if ( newState[ key ] ) {
```

Comment:
When would `newState[ key ]` not be defined?

Reply:
Given the shifts and changes in this PR no idea. Can only test and get back.

Reply:
Normally this should never be undefined since all the items to delete already should be in the state. I think I ended up with some weird queries while working on deleteEntityRecord or something ... Removed it as it looks useless.